CRYPTO_get_locking_callback is defined as null in openssl
so this is always true, but it fails to compile on musl 1.2.3
as casting from null to bool is invalid
--- a/src/SSL.cpp
+++ b/src/SSL.cpp
@@ -33,7 +33,7 @@
 	// If we detect that no locking callback is configured, we
 	// have to set it up ourselves to allow multi-threaded use
 	// of OpenSSL.
-	if (!CRYPTO_get_locking_callback()) {
+	if (true) {
 		SSLLocks::initialize();
 	}
 }
